### PR TITLE
fix(telegram): there should be a space between the first and last name

### PR DESCRIPTION
**Pull request by memset0**

### CODE REVIEW

File: adapters/telegram/src/utils.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const logger = new Logger('telegram')
 export const adaptUser = (data: Telegram.User): User => ({
   userId: data.id.toString(),
   username: data.username,
-  nickname: data.first_name + (data.last_name || ''),
+  nickname: data.last_name ? data.first_name + ' ' + data.last_name : data.first_name,
```
Comment (shigma) [inline adapters/telegram/src/utils.ts:10]: Maybe this is better?

```suggestion
  nickname: data.first_name + (data.last_name ? ' ' + data.last_name : ''),
```

Comment (memset0) [reply adapters/telegram/src/utils.ts:10]: emmm, I think it might be slower

Comment (shigma) [reply adapters/telegram/src/utils.ts:10]: Do you have concrete evidence that the above writing has significant performance issues?

Comment (memset0) [reply adapters/telegram/src/utils.ts:10]: idk, anyway it's up to you to decide

Comment (shigma) [reply adapters/telegram/src/utils.ts:10]: I think the suggested writing is clearer, easier to maintain and more similar to old version (based on the principle of minimal changes).

Comment (memset0) [reply adapters/telegram/src/utils.ts:10]: ok

**Review by shigma — APPROVED** [review]

**Pull request merged**